models/Season.ts

```diff
@@ -0,0 +1,53 @@
+import { Schema, model } from "mongoose";
+import { ISeason } from "../interfaces/season.interface";
+
+const PayrollTimeframeEnum = ["Weekly", "Biweekly", "Monthly"];
+
+const SeasonSchema = model<ISeason>(
+  "Season",
+  new Schema({
+    name: { type: String, required: true },
+    startDate: { type: Number, required: true },
+    endDate: { type: Number, required: true },
+    payroll_timeframe: {
+      type: String,
+      enum: PayrollTimeframeEnum,
+      required: true,
+    },
+    price: { type: Number, required: true },
+    status: { type: String, required: true, default: "Active" },
```

Comment:
let's create and enum and set it here, and replace the hardcoded value "Active"

Reply:
https://github.com/LangaraVoyagers/project-2-api/pull/8/commits/bda999083f5d89af1e9bfa8b076b19880029b043